src/components/LanguageSwitcher.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Dropdown, Label } from "@trussworks/react-uswds";
 import { useTranslation } from "react-i18next";
-import "./LanguageSwitcher.css";
+import "../styles/LanguageSwitcher.css";
```

Comment:
I think we want to use scss files, not css

Reply:
In the last PR, I mentioned: 

> I believe Justin said a while back to use CSS files for custom classes.

 We can address this with him if you'd like.

Reply:
Yes, I believe we would have all our themes in the settings.scss file and then we'd have one or more scss files with class names. We want to use scss so we can access variables. I don't think we'd want to limit ourselves to a regular css file